alf/algorithms/entropy_target_algorithm.py

```diff
@@ -61,6 +61,12 @@ class EntropyTargetAlgorithm(Algorithm):
        ``target_entropy`` (i.e., ``fast_stage_thresh`` in the code, which is the half
        of ``target_entropy`` if it is positive, and twice of ``target_entropy`` if
        it is negative.
+
+    ``EntropyTargetAlgorithm`` can be used to approximately reproduce the learning
+    of temperature in `Soft Actor-Critic Algorithms and Applications <https://arxiv.org/abs/1812.05905>`_.
+    To do so, you need to use the same ``target_entropy``, set ``skip_free_stage``
+    to True, and  set ``slow_update_rate`` and ``fast_update_rate`` to the 4
+    times of the learning rate for temperature.
```

Comment:
Is the number ``4`` an empirical number derived from the ICubWalk task? Maybe can add an explanation to it.

Reply:
Assuming the entropy fluctuate up and down for the most of time, one up swing will cause the log_alpha decrease by update_rate, one down swing will cause it to increase by 0.5*update_rate. Averaging over these two possibilities, we get the average change of log_alpha as 0.25 * update_rate.